testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -3623,6 +3639,22 @@ QUnit.module('Aria accessibility', {
             });
         });
     });
+
+    [
+        { attr: 'aria-label', value: 'Week 1' },
```

Comment:
Is it right that we can customize attributes for `week` cells?

Reply:
No, it's just accessibility test

Reply:
Yep wia cellTemplate